itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/CreateNewQuarkusConfigurationMavenTest.java

```diff
@@ -0,0 +1,121 @@
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/
+package org.jboss.tools.quarkus.integration.tests.project;
+
+import static org.junit.Assert.assertTrue;
+
+import org.eclipse.reddeer.common.wait.TimePeriod;
+import org.eclipse.reddeer.common.wait.WaitUntil;
+import org.eclipse.reddeer.eclipse.condition.ConsoleHasText;
+import org.eclipse.reddeer.eclipse.debug.ui.launchConfigurations.LaunchConfiguration;
+import org.eclipse.reddeer.eclipse.ui.console.ConsoleView;
+import org.eclipse.reddeer.eclipse.ui.navigator.resources.ProjectExplorer;
+import org.eclipse.reddeer.junit.runner.RedDeerSuite;
+import org.eclipse.reddeer.requirements.openperspective.OpenPerspectiveRequirement.OpenPerspective;
+import org.eclipse.reddeer.swt.condition.ShellIsAvailable;
+import org.eclipse.reddeer.swt.impl.button.PushButton;
+import org.eclipse.reddeer.swt.impl.menu.ContextMenuItem;
+import org.eclipse.reddeer.swt.impl.table.DefaultTableItem;
+import org.eclipse.reddeer.swt.impl.tree.DefaultTreeItem;
+import org.eclipse.reddeer.workbench.impl.shell.WorkbenchShell;
+import org.jboss.tools.quarkus.reddeer.perspective.QuarkusPerspective;
+import org.jboss.tools.quarkus.reddeer.wizard.CodeProjectTypeWizardPage;
+import org.jboss.tools.quarkus.reddeer.wizard.QuarkusWizard;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * https://issues.redhat.com/browse/JBIDE-27065
+ * New quarkus launch configuration integration test for Maven
+ * @author olkornii@redhat.com
+ */
+@OpenPerspective(QuarkusPerspective.class)
+@RunWith(RedDeerSuite.class)
+public class CreateNewQuarkusConfigurationMavenTest extends LaunchConfiguration {
+
+	public CreateNewQuarkusConfigurationMavenTest() {
+		super("Create New Quarkus Configuration Maven Test");
+	}
+
+	private static String MAVEN_PROJECT_NAME = "testProjectMaven";
+	private static ConsoleView consoleView;
+
+	@BeforeClass
+	public static void testNewNewQuarkusMavenProject() {
+		new WorkbenchShell().setFocus();
+
+		QuarkusWizard qw = new QuarkusWizard();
+		qw.open();
+		assertTrue(qw.isOpen());
+
+		CodeProjectTypeWizardPage wp = new CodeProjectTypeWizardPage(qw);
+		wp.setProjectName(MAVEN_PROJECT_NAME);
+		wp.setMavenProjectType();
+
+		qw.next();
+		qw.finish(TimePeriod.VERY_LONG);
+
+		assertTrue(new ProjectExplorer().containsProject(MAVEN_PROJECT_NAME));
+
+		consoleView = new ConsoleView();
+	}
+
+	@Test
+	public void testNewQuarkusMavenConfiguration() {
+		CreateNewQuarkusMavenConfiguration();
+		CheckNewQuarkusMavenConfiguration();
+		RunNewQuarkusMavenConfiguration();
```

Comment:
CheckNewQuarkusMavenConfiguration() and RunNewQuarkusMavenConfiguration() should be more independent: after checking configuration, you should close dialog with launch configurations. When running configuration you should open dialog with launch configurations.